dom5.ts

```diff
@@ -38,6 +38,15 @@ export function hasAttribute(element: Node, name: string): boolean {
   return getAttributeIndex(element, name) !== -1;
 }
 
+export function hasSpaceSeparatedAttributeValue(element: Node, name: string, value: string): boolean {
```

Comment:
There are only contrived reasons to have a predicate based on arbitrary delimiters.  YAGNI wins.

What I meant by "how functions are chosen" was more about asking why we have two styles of API in dom5.